cluster/build.gradle

```diff
@@ -41,7 +41,8 @@ dependencies {
     compile project(':snappy-spark:snappy-spark-hive_' + scalaBinaryVersion)
     compile project(':snappy-spark:snappy-spark-repl_' + scalaBinaryVersion)
     compile project(':snappy-spark:snappy-spark-streaming_' + scalaBinaryVersion)
-    compile project(':snappy-spark:snappy-spark-streaming-kafka-0.8_' + scalaBinaryVersion)
+    compile project(':snappy-spark:snappy-spark-streaming-kafka-0.10_' + scalaBinaryVersion)
+    compile project(':snappy-spark:snappy-spark-sql-kafka-0.10_' + scalaBinaryVersion)
```

Comment:
Have we checked if any of the existing customers are still using kafka 0.8 and depend on it? If so we should support both like Spark does. @sudsmenon was going to come back on that for existing customers.

Reply:
AFAIK, we don't have customers using 0.8, let @sudsmenon confirm though.

Reply:
I had a word with @sudsmenon and we are not having any customers using Kafka 0.8. So it should be fine to remove support for 0.8.